test/es-module/test-esm-exports.mjs

```diff
@@ -170,12 +170,7 @@ function assertIncludes(actual, expected) {
     '--experimental-conditional-exports',
     '/es-modules/conditional-exports.js',
     'Conditional exports',
-  ],
-  [
-    '--experimental-resolve-self',
-    '/node_modules/pkgexports/resolve-self.js',
```

Comment:
Is the file referenced here no longer needed? Looks like there's no references left after this deletion:

```console
$ ag resolve-self test/
test/es-module/test-esm-exports.mjs
163:    '--experimental-resolve-self',
164:    '/node_modules/pkgexports/resolve-self.js',
```

Reply:
It's referenced in `test/fixtures/node_modules/pkgexports/package.json`.

Reply:
(which is still used in the test cases here).

Reply:
Huh, not sure why my search didn't find that. Thanks!